### ISSUE TITLE
Can we get a dedicated forum for questions?

### ISSUE THREAD

**Issue by varunchopra** [tiangolo/fastapi#353]
Is it possible to get a dedicated forum for issues not related to bug reports and feature requests? I like reading the questions users have but it really messes up with my notifications. :/

For other projects, I like to keep track of questions other people have so I generally search a forum (google groups or something in a similar vein) but for FastAPI I've had to put the repo on watch for bug reports so that I can update my app accordingly and this is becoming a mess.

If needed, I can volunteer to set something up for this.

**Comment by euri10** [comment]
Have you tried the gitter chat?

**Comment by tiangolo** [comment]
@chopraaa questions are generally here (as this one).

Some bugs end up being actually questions, not bug reports.

And more general conversation happens on the Gitter chat here: https://gitter.im/tiangolo/fastapi

If you only want to know when there are new features, you can watch only the releases. But if you want to receive notifications for questions, I guess this would be the best place.

**Comment by chopraaa** [comment]
Gitter is a blink-and-you-miss-it scenario for me. I'd rather have threads catered to specific questions so I can remember where I read about the particular issue instead of scrolling through chat.

Just a suggestion. Thanks.

**Comment by tiangolo** [comment]
Yep, I agree that Gitter is a bit chaotic many times, and not necessarily very searchable.

For general questions, GitHub issues would probably be the right/equivalent place.

I guess a forum would probably have a comparable amount of notifications with questions, reports, etc.
